pkg/querier/frontend/frontend.go

```diff
@@ -12,6 +13,7 @@ import (
 	"path"
 	"strings"
 	"sync"
+	"sync/atomic"
```

Comment:
Let's use `connectedClients *atomic.Int32` instead. It makes it clear that this is accessed atomically, and it also avoids possible alignment issues (although that's not relevant for 32-bit values I think).

Reply:
good call.  updating.